bin/tictactoe

```diff
@@ -1,3 +1,38 @@
 #!/usr/bin/env ruby
 
 require_relative '../config/environment'
+
+puts "Welcome to Tic-Tac-Toe."
+puts 'Enter "0" for computer game; Enter "1" for a 1-player game; Enter "2" for a 2-player game'
+input = gets.chomp
+if input.downcase == "wargames"
+  input = 4
+end
+input = input.to_i
+
+
+case input
+when 0
+  computer_game = Game.new(Players::Computer.new("X"), Players::Computer.new("O"), Board.new)
+  computer_game.play
+when 1
+  single_player = Game.new(Players::Human.new("X"), Players::Computer.new("O"), Board.new)
+  single_player.play
+when 2
+  puts "Player X, please go first."
+  two_player = Game.new(Players::Human.new("X"), Players::Human.new("O"), Board.new)
+  two_player.play
+when 4
+  count = 0
+
+  100.times do
+    wargame = Game.new(Players::Computer.new("X"), Players::Computer.new("O"), Board.new)
+    wargame.play
+    if wargame.won?
+      count += 1
+    end
+  end
+  puts "The game was won #{count} times!"
+else
+  puts "Please enter 0-2"
+end
```

Comment:
Prefer to encapsulate CLI logic into a class. For example, the only code in your bin executable should be:

```ruby
#!/usr/bin/env ruby
       
require_relative '../config/environment'

TicTacToeCLI.new.call
```

And then you'd have a file: `lib/tic_tac_toe_cli.rb` with:

```ruby
class TicTacToeCLI
  def initialize
  end

  def call
   # Your game logic
  end
end
```

The code in your bin would generally just go in `call`.